public/docs/swagger.yaml

```diff
@@ -414,644 +412,468 @@ definitions:
     Crop:
         type: object
         properties:
-            IDX_Crop:
-                type: number
-                description: to be updated
-                example: 2322356
             IDX_Property:
                 type: number
```

Comment:
@thinkking can you confirm with @nick22891 the attribute type for the resource schema. This should then be reflected in the documentation pull request.